apps/src/blockly/eventHandlers.ts

```diff
@@ -117,3 +122,78 @@ export function reflowToolbox() {
     modalWorkspace?.getFlyout()?.reflow();
   }
 }
+export function updateBlockLimits(event: Abstract) {
+  // This check is to update bubbles that show block limits whenever
+  // blocks on the main workspace are updated.
+  if (
+    event.type !== Blockly.Events.BLOCK_CHANGE &&
+    event.type !== Blockly.Events.BLOCK_MOVE &&
+    event.type !== Blockly.Events.BLOCK_CREATE &&
+    event.type !== Blockly.Events.THEME_CHANGE
```

Comment:
why do we need to check theme change events?

Reply:
If you switch in/out of high contrast mode, the size of all `blocklyText` changes, including the on the indicators. Will leave a comment!